src/pipx/constants.py

```diff
@@ -30,6 +30,48 @@ def is_windows() -> bool:
 WINDOWS: bool = is_windows()
 
 
+# If we are running under the Windows embeddable distribution,
+# venv isn't available (and we probably don't want to use the
+# embeddable distribution as our applications' base Python anyway)
+# so we try to locate the system Python and use that instead.
+
+# This code was copied from https://github.com/uranusjr/pipx-standalone
+# which uses this technique to build a completely standalone pipx
+# distribution
+
+
+def _find_default_python():
```

Comment:
If this is only used on windows, suggest changing the name to contain `windows`, something like `_find_default_python_windows()`.

Reply:
No problem, done.